com.orsan.Turtlico.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "com.orsan.Turtlico",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "turtlico",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
```

Comment:
These two dconf lines also should be removed. GLib >=2.60.6 stores settings locally, without host access.